client/src/reducers.tsx

```diff
@@ -42,6 +45,10 @@ export function gameStatsReducer(state = initialGameStats, action: any) {
             return { ...state, roundInProgress: action.payload}
         case 'gameStats/toggleGameStarted': 
             return { ...state, gameStarted: action.payload}
+        case 'gameStats/setHealth': 
```

Comment:
Is this still being used if you also have a health state itself? Should health and points just be held in the player structure since they're singular?

Reply:
Oh never mind, health is like scores where it will be a huge structure of all player healths, so we should keep it separate from player. I think we can still remove it from gameStats though.